src/index.help.ts

```diff
@@ -74,6 +90,11 @@ export const deployment = async (
     SHOPIFY_THEME_PREVIEW_URL: themePreviewUrl,
   });
 
-  const message = `:tada: Shopify theme has been deployed to theme id '${shopifyThemeId}' at '${SHOPIFY_AUTH.storeUrl}'. The theme can be previewed at: ${themePreviewUrl}`;
-  await createReplaceComment(message, UNIQUE_HIDDEN_COMMENT_STRING, shopifyThemeId, GITHUB_AUTH);
+  const message = `:tada: '${SHOPIFY_AUTH.storeUrl}' preview at: \n <${themePreviewUrl}>`;
```

Comment:
There is benefit to keeping the theme id in the comment as a separate item, so I think this should be reverted to what it currently is

Reply:
@matthew-petrie 
Just wanted to simplify the comment - but yeah, the ID could be as a separate item.
Anyway:
From my own exprience there should be also Store url, since in some cases, devs are working on different environments within the same theme:
E.g. The company, i am working for, uses different stores for Staging, Production, sometimes even more ;)

Reply:
@matthew-petrie i have reviewed, actually there still exists ID as a separate item - in hidden comment, i can't see any benefits of keeping them in the visible part additionally since preview url contains also themeid
What do you think?